hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalWanPublisherStatsImpl.java

```diff
@@ -121,8 +117,8 @@ public JsonObject toJson() {
         root.add("totalPublishLatencies", totalPublishLatency);
         root.add("totalPublishedEventCount", totalPublishedEventCount);
         root.add("outboundQueueSize", outboundQueueSize);
-        root.add("paused", paused);
-        root.add("stopped", stopped);
+        root.add("paused", isPaused());
```

Comment:
@emre-aydin @mesutcelik what are the guarantees of the JSON format for objects sent to mancenter? Do we need to have it backwards compatible? Is it public and can it be changed?

Reply:
It's not public, it can be changed. We need to have it backward compatible for one minor version. If this is to be released as 3.9.4-c1, MC 3.9.x and MC 3.10.x needs to be able to work with this. You can add extra fields which we will use in an upcoming MC 3.10.x release. Removing the field needs to be delayed until HZ 3.11 where we can adjust the codebase to not expect the field.